internal/node/launcher.sh

```diff
@@ -330,6 +343,23 @@ _int() {
   kill -INT "${child}" 2>/dev/null
 }
 
+_exit() {
+  EXIT_CODE=$?
+
+  if [[ "$EXIT_CODE" != 0 ]]; then
+    if [ ${STDOUT_CAPTURE_IS_NOT_AN_OUTPUT} = true ]; then
+      cat "$STDOUT_CAPTURE"
```

Comment:
I debated doing a breaking change here of `cat`ing the output even if it is being captured. What do you think?

Reply:
I think this is fine, conceptually if the output is "captured" then I can see the case for it not making it to stdout. No ones opened an issue so far saying that they'd like to see the output of captured outs.